Base/home/anon/Terminal.ini

```diff
@@ -0,0 +1,2 @@
+[Window]
+Opacity=150
```

Comment:
I think the default should be 255 opacity (or 100% or however we express it)
Semitransparent windows look neat but they are still so slow ;)